### PR TITLE
1.3.0

### PR DESCRIPTION
**Pull request by prashanthpx**
**What this PR does / why we need it**:
Support for px-backup 2.0.0 in aws market place

**Which issue(s) this PR fixes** (optional)

**Special notes for your reviewer**:

### CODE REVIEW

File: px-central/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- storage
+- persistent disk
+- pvc
+- portworx
+- px-central
+- lighthouse
+name: px-central
+sources:
+- https://github.com/portworx/helm/tree/master/charts/px-central
+version: 1.3.0
```
Comment (diptiranjanpx) [inline px-central/Chart.yaml:16]: Version should be 2.0.0

File: px-central/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+appVersion: 1.3.0
```
Comment (diptiranjanpx) [inline px-central/Chart.yaml:17]: Appversion is also 2.0.0.

File: stable/index.yaml [hunk 3]
```diff
@@ -38,6 +38,28 @@ entries:
...
+    - persistent disk
+    - pvc
+    - portworx
+    - px-central
+    - lighthouse
+    name: px-central
+    sources:
+    - https://github.com/portworx/helm/tree/master/charts/px-central
+    urls:
+    - https://raw.githubusercontent.com/portworx/helm/1.3.0/stable/px-central-1.3.0.tgz
```
Comment (diptiranjanpx) [inline stable/index.yaml:63]: index wrt master like
helm repo index --url https://raw.githubusercontent.com/portworx/aws-helm/master/stable/ .

**Review by diptiranjanpx — COMMENTED** [review]
Also helm package should create px-central-2.0.0 tar bundle

File: stable/index.yaml (hunk 3 above)
Comment (prashanthpx) [reply stable/index.yaml]: fixed

File: px-central/Chart.yaml (hunk 1 above)
Comment (prashanthpx) [reply px-central/Chart.yaml]: fixed

File: px-central/Chart.yaml (hunk 2 above)
Comment (prashanthpx) [reply px-central/Chart.yaml]: fixed

**Review by diptiranjanpx — APPROVED** [review]

**Pull request merged**